### ISSUE TITLE
jsPsych Response Data

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1283]
# TL;DR
JsPsych will need to update the participant's data via lookit's API. 

# Narrative
The participant's data is stored in the response table.  Once the response has been created, then it'll need to be updated as the participant works through the study.  This can be done in the [`on_data_update` hook in jsPsych](https://www.jspsych.org/7.3/overview/events/#on_data_update).  

# Acceptance Criteria
- [ ] Create a response object before the participant starts
- [ ] Update response data as it's available  
- [ ] Lookit API should be leveraged through a library within it's own module. 
- [ ] Tests

# Implementation Notes
Quick search found a few libraries that can be used to interact with a RESTful API; Axios, node-fetch, and got.  A home rolled solution is possible with built-in JavaScript functions, but I am just lead to believe some features aren't available natively.  Such as retry.
